contrib/mesos/pkg/node/node.go

```diff
@@ -36,11 +36,10 @@ const (
 )
 
 // Create creates a new node api object with the given hostname and labels
-func Create(client *client.Client, hostName string, labels map[string]string) (*api.Node, error) {
+func Create(client *client.Client, hostName string, slaveAttrLabels, annotations map[string]string) (*api.Node, error) {
```

Comment:
It looks like the expectation is that the `annotations` map must not be nil. If that's the case, then we should probably document it. If that's NOT the intent, then why do we pass zero-values in when calling this func (and Update())?